workbench-core/example/infrastructure/integration-tests/tests/users/createUser.test.ts

```diff
@@ -43,6 +43,13 @@ describe('userManagement create user integration test', () => {
   });
 
   it('should return a 400 error when a user with the provided email already exists', async () => {
+    try {
+      // this test fails when the user doesn't already exist.
+      await adminSession.resources.users.create(user);
+    } catch (error) {
+      // could throw if user already exists
+      console.warn(error);
+    }
```

Comment:
Addressed this in #619 so you'll probably have a merge conflict

Reply:
Your way addresses the issue the way I originally fixed it to work in my environment. I changed it late in the game to make it a little more robust as it seemed you may have been testing on a dirty environment. I reverted to your less robust version as I prefer cleaner environments.